Explorer/Assets/DCL/InWorldCamera/CameraReelGallery/Components/StorageProgressBar.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using TMPro;
+using UnityEngine;
+
+namespace DCL.InWorldCamera.CameraReelGallery.Components
+{
+    public class StorageProgressBar : MonoBehaviour
+    {
+        private const float MIN_VALUE = 0f;
+        private const float MAX_VALUE = 100f;
+
+        private const string LABEL_PLACEHOLDER = "Storage {0}/{1} photos taken";
+
+        [Range(MIN_VALUE, MAX_VALUE)] public float valuePercentage;
+
+        [Space]
+        [Header("Configuration")]
+        public float? MaxRealValue;
```

Comment:
Try to avoid open fields - it violates encapsulation. Considering how you use it you can integrate these values into `SetPercentageValue` method, and remove redundant checks